wakeup-server/tests/test_interactive_devices_exception.py

```diff
@@ -0,0 +1,46 @@
+import pytest
+
+
+def test_empty_name(client):
+    response = client.post("/interactive_devices", json={"name": "", "type": "SOUND"})
+    obj = response.get_json()
+    with pytest.raises(TypeError) as e:
+        name = obj['name']
+    msg = e.value.args[0]
+    assert 'NoneType' in msg
+
+
+def test_too_long_name(client):
+    response = client.post("/interactive_devices", json={"name": "a"*1000, "type": "SOUND"})
+    obj = response.get_json()
+    with pytest.raises(Error) as e:
+        name = obj['name']
```

Comment:
Here you need to test if the sends back the right error message. Because there's nothing impemented yet, just write one that makes sense